cadquery/cq.py

```diff
@@ -2715,14 +2715,15 @@ def combine(self, clean=True):
 
         return self.newObject([s])
 
-    def union(self, toUnion=None, clean=True):
+    def union(self, toUnion=None, clean=True, glue=False, tol=None):
         """
         Unions all of the items on the stack of toUnion with the current solid.
         If there is no current solid, the items in toUnion are unioned together.
         
         :param toUnion:
         :type toUnion: a solid object, or a CQ object having a solid,
         :param boolean clean: call :py:meth:`clean` afterwards to have a clean shape
+        :param boolean glue: use a faster gluing mode for non-overlapping shapes
         :raises: ValueError if there is no solid to add to in the chain
```

Comment:
Seems to be missing `tol` parameter in doc string. `tol` being `None` won't break anything? Does it just let OCCT auto-select the tolerance in that case?

Reply:
Good catch. Adding the docstring.